src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/highlightablebutton/HighlightableButton.kt

```diff
@@ -0,0 +1,53 @@
+package org.wycliffeassociates.otter.jvm.app.widgets.highlightablebutton
+
+import com.jfoenix.controls.JFXButton
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.beans.property.SimpleBooleanProperty
+import javafx.scene.paint.Color
+import tornadofx.*
+
+class HighlightableButton(
+        private val primaryColor: Color,
+        private val secondaryColor: Color,
+        private val hasBorder: Boolean,
+        private val isHighlighted: SimpleBooleanProperty,
+        private val icon: MaterialIconView? = null
```

Comment:
I didn't want to put this here, but the fill has to be applied directly on the MaterialIconView (applying the fill to the graphic Node does not work.) All of our use cases have icons, but I made this a nullable parameter.